code/modules/mob/living/carbon/human/human.dm

```diff
@@ -371,18 +322,14 @@
 
 
 /mob/living/carbon/human/Topic(href, href_list)
-
-	if (href_list["refresh"])
-		if((machine)&&(in_range(src, usr)))
-			show_inv(machine)
-
-	if (href_list["mach_close"])
+	if (href_list["mach_close"]) // This is horrible.
 		var/t1 = text("window=[]", href_list["mach_close"])
 		unset_machine()
 		src << browse(null, t1)
 
 	if(href_list["item"])
-		handle_strip(href_list["item"],usr)
+		log_runtime(EXCEPTION("Warning: human/Topic was called with item [href_list["item"]], but the item Topic is deprecated!"))

```

Comment:
I didn't see anything that could call this, but I'm paranoid of magic bullshit with Topic() calls after seeing some horrible, horrible things during the last few TGUI ports, so fuck it, runtimes.